### ISSUE TITLE
Does it work in Roundcube 1.0.5?

### ISSUE THREAD

**Issue by mark4412** [sblaisot/automatic_addressbook#1]
I tried installing it on roundcube 1.0.5 but when I ran the sql script it gave me this error : 

errno: 150
InnoDB
Supports transactions, row-level locking, and foreign keys

I'm wondering if maybe it's because there's not a 'user_id_fk_collected_contacts' table? Or maybe it just doesn't support Roundcube 1.0.5? But I only see the requirement of Roundcube > 0.4 so I thought it should work.

Thanks in advance. Any help would be greatly appreciated.

**Comment by theqkash** [comment]
bump

**Comment by wlp7s0** [comment]
I have roundcube 1.1.4. There was no errors while installation, but it doesn't work. Mysql table is empty but plugin is listed in plugin list. That's sad. 
In fact, it shows Deprecation notice while install. May be that is the problem?
"Deprecation Notice: The Composer\Package\LinkConstraint\VersionConstraint class is deprecated, use Composer\Semver\Constraint\Constraint instead. in phar:///var/www/mail/composer.phar/src/Composer/Package/LinkConstraint/VersionConstraint.php:17
Deprecation Notice: The Composer\Package\LinkConstraint\LinkConstraintInterface interface is deprecated, use Composer\Semver\Constraint\ConstraintInterface instead. in phar:///var/www/mail/composer.phar/src/Composer/Package/LinkConstraint/LinkConstraintInterface.php:17"

**Comment by hyperlinked** [comment]
wlp7s0, I installed it manually and ran the SQL script through phpMyAdmin. It works fine in 1.1.4, but I don't use Composer so I don't know if it has any problems with an install done that way.

**Comment by hyperlinked** [comment]
mark4412, you need the database table for this to work. Without the table, the contacts have nowhere to be stored. If you're doing a manual install, make sure you run the SQL script to create the table you need as stated in the instructions. I don't know what your DB might be called, but mine is simply called "roundcube". Based on the naming convention of your DB table, I'd guess yours might be named user_id_fk_roundcube.

**Comment by dapphp** [comment]
I can also confirm this works on 1.1.4.  I installed via Composer and the DB table seemingly was created automatically.  Tested and recipients are automatically added and autocomplete works.

**Comment by alexdom** [comment]
I have issues with Rouncube 1.1.2. 
  I use PSQL. The tables are created using SQL script. But after installation I found that addresses are not collected. Even after granting all privileges to **`roundcube`** as owner of the **`collected_contacts`** table. Role **`roundcube`** is the owner of the **`rouncubemail`** database and all its tables now. 
  So after sending emails to certain addresses I was checking the table **`collected_contacts`** in PSQL every time with command:    `TABLE collected_contacts;`. No luck. The table remained empty. 
  So I tried to `INSERT` some data in table manually: 
   `INSERT INTO collected_contacts VALUES (1, 1, now(), 0, 'Nickname', 'some@email.com', 'Name', 'Lastname');`
  That helped and data appeared in the table. But in Roundcube interface in `"Automatically Collected"` address book - was total emptiness. Also, when trying to add record via Roundcube interface, it gives an error:      `An error occurred while saving.` That looks like a DB access issue. So I don't know how you guys made it to work.
